addons/featured-dangos-v2/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Featured dangos",
+  "description": "An Apil Fools joke from 2023 that adds a row to the front page for dango-related projects.",
```

Comment:
```suggestion
  "description": "An April Fools joke from 2023 that adds a row to the front page for dango-related projects.",
```

Reply:
Should we also specify that this was originally added by Scratch Addons?